Source/Core/AudioCommon/AlsaSoundStream.h

```diff
@@ -21,12 +21,12 @@ class AlsaSound final : public SoundStream
 #if defined(HAVE_ALSA) && HAVE_ALSA
 public:
   AlsaSound();
+  ~AlsaSound();
```

Comment:
This can have an `override` placed on it. Ditto for the other added SoundStream subclasses that have a destructor added. This seems to have been done for the XAudio2 backend, but not the others.

Reply:
Fixed, good catch.